source/loader/ze_loader_internal.h

```diff
@@ -26,6 +26,18 @@
 #include "spdlog/spdlog.h"
 namespace loader
 {
+    ///////////////////////////////////////////////////////////////////////////////
+    /// @brief Driver Type Enumerations
+    typedef enum _zel_driver_type_t
+    {
+        ZEL_DRIVER_TYPE_DISCRETE_GPU= 0,          ///< The driver has Discrete GPUs only
+        ZEL_DRIVER_TYPE_GPU = 1,                  ///< The driver has Heterogenous GPU types
```

Comment:
I see that iGPU can be assumed as slower than dGPU  but why assuming that dGPU only  driver is better than dGPU + iGPU ?

Reply:
This is an assumption based on an arch review. This is not guaranteed to always be true which is why customers should always ensure the right device is being used in their workloads and not always assume device 0 is the best.